components/avatar.css

```diff
@@ -0,0 +1,49 @@
+.img-round{
+    border-radius: 50%;
+    object-fit: cover;
+}
+
+.img-responsive{
+    width: 100%;
+    height: 100%;
+    
+}
+
+.avatar{
+    border-radius: 50%;
+}
+
+.avatar-text{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #1D53FF;
+    color: white;
+    font-weight: bold;
```

Comment:
You can try to use css variables here

Reply:
Good catch there. Refactored and pushed. Thanks Ankur.

Reply:
Good catch there. Refactored ,thanks Ankur.